src/init.cpp

```diff
@@ -1633,7 +1635,7 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
         while (!fHaveGenesis) {
             condvar_GenesisWait.wait(lock);
```

Comment:
Maybe worth changing to `condvar_GenesisWait.wait(lock, []{ return fHaveGenesis; });` and dropping the loop.

Reply:
Done.